cp/cp_config.h

```diff
@@ -12,8 +12,8 @@
 #include <sys/queue.h>
 #include "stdbool.h"
 
-#define CP_CONFIG_ETC_PATH		"../config/app_config.cfg"
-#define CP_CONFIG_OPT_PATH		"/tmp/app_config.cfg"
+#define CP_CONFIG_ETC_PATH		"/etc/cp/config/app_config.cfg"
```

Comment:
I am thinking how to get rid of this. couple of option I have

Take a argument in the ngic_controlplane. This argument will tell alternate config file path. It should be /etc/cp/config/app_config.cfg . If this argument is not present then ngic_controlplane will put watch triggers on ../config/app_config.cfg file.

This will make sure that code not having any etc path. Etc path will come as an argument to the spgw.
@krsna1729  Do you agree with this approach ?